schema/artwork/index.js

```diff
@@ -439,6 +439,16 @@ const ArtworkType = new GraphQLObjectType({
       manufacturer: markdown(),
       series: markdown(),
       meta: Meta,
+      related: {
+        type: new GraphQLList(Artwork.type),
+        args: {
```

Comment:
doesn't look like this allows for paging/size: https://github.com/artsy/gravity/blob/a385e1e2b3fd31c85a30b0b529f30838803ce11b/app/api/v1/related_endpoint.rb#L268-L277

and the index it reads: https://github.com/artsy/gravity/blob/a385e1e2b3fd31c85a30b0b529f30838803ce11b/app/models/search/cf_artwork_similarity_index.rb